itwin-platform-access/imodels-access-backend/src/interface-adapters/ClientToPlatformAdapter.ts

```diff
@@ -51,11 +51,11 @@ export class ClientToPlatformAdapter {
       });
 
     return {
-      containerId: containerAccessInfo.container,
-      sasToken: containerAccessInfo.sas,
-      accountName: containerAccessInfo.account,
-      dbName: containerAccessInfo.dbName,
-      storageType: "azure?sas=1"
+      containerId: directoryAccessInfo.baseDirectory,
+      sasToken: directoryAccessInfo.storageType === "azure" ? directoryAccessInfo.azure!.sasToken : directoryAccessInfo.google!.authorization,
```

Comment:
Maybe we should throw if directoryAccessInfo.storageType is not "azure" or "gcp"? To avoid confusion if we add another storageType in the future.

Reply:
Added throwing for storageType specifically, haven't yet added validation for other properties.